websites/B/BeatLeader/metadata.json

```diff
@@ -13,8 +13,8 @@
 		"replay.beatleader.xyz",
 		"royale.beatleader.xyz"
 	],
-	"version": "1.1.1",
-	"logo": "https://i.imgur.com/TWPjaYR.png",
+	"version": "1.2.0",
+	"logo": "https://i.imgur.com/PhIU3mx.png",
```

Comment:
Please make the logo 512x512

Reply:
> Please make the logo 512x512

The guidelines says "The image is required to have a **minimum** resolution of 512x512 pixels."
I will change it but, if this has changed to a hard requirement of 512x512 would you be so kind as to update the guidelines?